src/ui/public/chrome/directives/kbn_chrome.js

```diff
@@ -44,7 +47,9 @@ export default function (chrome, internals) {
         onRouteChange();
 
         // and some local values
-        chrome.httpActive = $http.pendingRequests;
+        $scope.$watch(() => $http.pendingRequests, (pendingHttpRequests) => {
```

Comment:
Can we just use `$watchCollection` here instead?  That will watch for changes to the actual array reference or to any of the request references, but won't deeply watch all properties of the requests themselves.